internal/infra/turnstile/verifier.go

```
@@ -0,0 +1,92 @@
+	}
+	req.Header.Set("Content-Type", "application/x-www-form-urlencoded")
+
+	client := v.client
+	if client == nil {
+		client = http.DefaultClient
+	}
+	resp, err := client.Do(req)
+	if err != nil {
+		return errors.Join(usecase.ErrInternalError, err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)